packages/storybook/stories/Dialpad/Dialpad.stories.tsx

```diff
@@ -3,36 +3,61 @@
 
 import { Dialpad as DialpadComponent, useTheme } from '@azure/communication-react';
 import { mergeStyles } from '@fluentui/react';
-import { Canvas, Description, Heading, Props, Title } from '@storybook/addon-docs';
+import { Canvas, Description, Heading, Props, Subheading, Title } from '@storybook/addon-docs';
 import { Meta } from '@storybook/react/types-6-0';
 import React from 'react';
 import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
 import { COMPONENT_FOLDER_PREFIX } from '../constants';
 import { hiddenControl } from '../controlsUtils';
 import { CustomDialpadExample } from './snippets/CustomDialpad.snippet';
 import { DialpadExample } from './snippets/Dialpad.snippet';
+import { DialerExample } from './snippets/DialpadDialer.snippet';
 
 const CustomDialpadText = require('!!raw-loader!./snippets/CustomDialpad.snippet.tsx').default;
 const ExampleDialpadText = require('!!raw-loader!./snippets/Dialpad.snippet.tsx').default;
+const DialerExampleText = require('!!raw-loader!./snippets/DialpadDialer.snippet.tsx').default;
 
 const getDocs: () => JSX.Element = () => {
   /* eslint-disable react/no-unescaped-entities */
   return (
     <>
-      <SingleLineBetaBanner version={'1.3.2-beta.1'} topOfPage={true} />
+      <SingleLineBetaBanner topOfPage={true} />
       <Title>Dialpad</Title>
+      <Heading>Dialpad modes</Heading>
       <Description>
-        Component to render a Dialpad. This component allows numbers and +, *, # input by clicking on dialpad or using
-        keyboard
+        The Dialpad component is versitile in its usage. Like on a physical phone the dialpad can have many different
+        jobs. Our Dialpad component is designed to fit the different scenarios that can be encountered in calling
+        experiences.
+      </Description>
+      <Subheading>Dialer Mode</Subheading>
+      <Description>
+        This is the default mode of the Dialpad component. In this mode you are able to enter and edit numbers in the
+        Dialpad's input box. The typical scenario to use this mode is when you are dialing a number to call, or to dial
+        in a new participant into a ongoing call.
+      </Description>
+      <Canvas mdxSource={DialerExampleText}>
+        <DialerExample />
+      </Canvas>
+      <Subheading>DTMF Mode</Subheading>
+      <Description>
+        This mode is for sendinf DTMF tones when in a call. These tones are for controlling the bot or other service
+        that that that you might encounter. Each tone is represented by a sound that is played when pressing each key.
+        You can disable these sounds by using the `disableDtmfPlayback` property on the component. In this mode the
+        input box in the dialpad is hidden since you are not able to edit DTMF tones that you have sent.
```

Comment:
```suggestion
        This mode is for sending DTMF tones when in a call. These tones are used for controlling bots or other services
        that you might encounter. Each tone is mapped to a sound that is played when pressing each key.
        You can disable these sounds by using the `disableDtmfPlayback` property on the component. In this mode, the
        input box in the dialpad is hidden since you are not able to edit DTMF tones that are sent.
```